### PR TITLE
docs: Removed version column in 4.0

### PR DESCRIPTION
**Pull request by lambGirl**
### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [X] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link
ref: #19815 

### 💡 Background and solution
ref: #19815 

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |   Removed version column in 4.0        |
| 🇨🇳 Chinese |    删除相关版本号       |

### ☑️ Self Check before Merge

- [X] Doc is updated/provided or not needed
- [X] Demo is updated/provided or not needed
- [X] TypeScript definition is updated/provided or not needed
- [X] Changelog is provided or not needed

-----
[View rendered components/grid/index.en-US.md](https://github.com/likeDoMine/ant-design/blob/lamia-antd4.0/components/grid/index.en-US.md)
[View rendered components/grid/index.zh-CN.md](https://github.com/likeDoMine/ant-design/blob/lamia-antd4.0/components/grid/index.zh-CN.md)
[View rendered components/menu/index.en-US.md](https://github.com/likeDoMine/ant-design/blob/lamia-antd4.0/components/menu/index.en-US.md)
[View rendered components/menu/index.zh-CN.md](https://github.com/likeDoMine/ant-design/blob/lamia-antd4.0/components/menu/index.zh-CN.md)
[View rendered components/page-header/index.en-US.md](https://github.com/likeDoMine/ant-design/blob/lamia-antd4.0/components/page-header/index.en-US.md)
[View rendered components/page-header/index.zh-CN.md](https://github.com/likeDoMine/ant-design/blob/lamia-antd4.0/components/page-header/index.zh-CN.md)
[View rendered components/pagination/index.en-US.md](https://github.com/likeDoMine/ant-design/blob/lamia-antd4.0/components/pagination/index.en-US.md)
[View rendered components/pagination/index.zh-CN.md](https://github.com/likeDoMine/ant-design/blob/lamia-antd4.0/components/pagination/index.zh-CN.md)
[View rendered components/steps/index.en-US.md](https://github.com/likeDoMine/ant-design/blob/lamia-antd4.0/components/steps/index.en-US.md)
[View rendered components/steps/index.zh-CN.md](https://github.com/likeDoMine/ant-design/blob/lamia-antd4.0/components/steps/index.zh-CN.md)
[View rendered components/typography/index.en-US.md](https://github.com/likeDoMine/ant-design/blob/lamia-antd4.0/components/typography/index.en-US.md)
[View rendered components/typography/index.zh-CN.md](https://github.com/likeDoMine/ant-design/blob/lamia-antd4.0/components/typography/index.zh-CN.md)

### CODE REVIEW

**Comment by yoyo837** [comment]
https://github.com/ant-design/ant-design/issues/19815#issuecomment-555867885

**Comment by zombieJ** [comment]
Code conflict

File: components/_util/colors.ts [hunk 1]
```diff
@@ -18,5 +18,5 @@ export const PresetColorTypes = tuple(
   'lime',
 );
 
-export type PresetColorType = (typeof PresetColorTypes)[number];
-export type PresetStatusColorType = (typeof PresetStatusColorTypes)[number];
+export type PresetColorType = typeof PresetColorTypes[number];
```
Comment (zombieJ) [inline components/_util/colors.ts]: Not change non-md files

File: components/input/ClearableLabeledInput.tsx [hunk 2]
```diff
@@ -16,7 +16,7 @@ export function hasPrefixSuffix(props: InputProps | ClearableInputProps) {
  */
 interface BasicProps {
   prefixCls: string;
-  inputType: (typeof ClearableInputType)[number];
```
Comment (zombieJ) [inline components/input/ClearableLabeledInput.tsx]: Not change non-md files

**Pull request closed without merging**